test/ajax.spec.js

```diff
@@ -6,19 +6,68 @@ var forEachOwnProperties = require('../collection/forEachOwnProperties');
 var ajax = require('../ajax/index.js'); // Test for transpiled file
 // var ajax = require('../ajax')['default']; // Test for ES6 Module
 
+var mock = {
+  open: jest.fn(),
+  send: jest.fn().mockImplementation(function() {
+    this.readyState = 4;
+    this.onreadystatechange();
+  }),
+  setRequestHeader: jest.fn(),
+  getResponseHeader: jest.fn().mockImplementation(function(key) {
+    return this.responseHeaders && this.responseHeaders[key];
+  }),
+  getAllResponseHeaders: jest.fn().mockImplementation(function() {
+    var result, i, keys;
+
+    if (this.responseHeaders) {
+      result = '';
+      keys = Object.keys(this.responseHeaders);
+
+      for (i = 0; i < keys.length; i += 1) {
```

Comment:
`key.length`도 변수에 담아서 사용하면 좋을 것 같아요.

Reply:
```suggestion
      var length = keys.length;
      for (i = 0; i < length; i += 1) {
```
위처럼 변수에 담아 사용하는 것으로 변경하겠습니다.